mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
```

Comment:
The function signature is incomplete. It should include the entire function body to ensure proper review. Please provide the complete function implementation.